HelloPi/README.md

```diff
@@ -0,0 +1,70 @@
+
+# HelloPi
+
+A simple HelloFX application with Java 11+, JavaFX 11+ and GraalVM that runs on HotSpot or as a native image on Desktop and Embedded devices (Raspberry Pi).
+
+## Documentation
+
+Read about getting started with JavaFX on Embedded [here](https://docs.gluonhq.com/#platforms_embedded)
+
+Note that Gluon applications can run on JVM/HotSpot on ARM devices (32/64 bits) but native images currently can be created only for AArch64 (64 bits).
+
+## Quick Instructions
+
+We use [GluonFX plugin](https://docs.gluonhq.com/) to build a native image for platforms including desktop, embedded, android and iOS.
+Please follow the prerequisites as stated [here](https://docs.gluonhq.com/#_requirements).
+
+While running this sample on a JVM can be done directly from the Raspberry Pi (both 32 and 64 bits), the following instructions are mainly describing how to cross-compile and remote deploy a native image from a desktop Linux machine (x86_64) to a Raspberry Pi (64 bits, AArch64).
+
+### Desktop
+
+Testing, fixing, and development in general, is best done on desktop. Therefore, before deploying to the Raspberry Pi, you can run the application on a JVM on your desktop (x86_64) machine:
+
+    mvn gluonfx:run
+
+Then run the application and explore all scenarios to generate config files for the native image with:
+
+    mvn gluonfx:runagent
+
+Build a native image using:
+
+    mvn gluonfx:build
+
+Run the native image app:
+
+    mvn gluonfx:nativerun
+   
+### Embedded (Raspberry Pi 32/64 bits)
```

Comment:
We may want to update the heading here to differentiate it from the next section. For example: "Embedded - JVM" or "Embedded - JRE". And the next section can be updated to "Embedded - Native", or something similar.

Reply:
Makes sense, done.